gui.py

```
@@ -0,0 +1,550 @@
+        self._run(args)
+
+    def _run_install(self):
+        ver = self._val(self.inst_ver)
+        if not ver: self._warn("Version or .swu path is required."); return
+        if not self._confirm(f"Install '{ver}' on the device?"): return
+        self._run(["install", ver])
+
+    def _run_backup(self):
+        args = ["backup"]
+        out = self._val(self.bk_out)
+        if out: args += ["-o", out]
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)